skin/frontend/base/default/js/oyst/oneclick.js

```diff
@@ -11,7 +11,68 @@
 /**
  * OneClick Customs JS
  */
+"use strict";
 
+/**
+ * Main function to init 1-Click
+ *
+ * @param productId
+ * @param oneClickUrl
+ * @param isProductAddtocartFormValidate
+ */
+function oystOneClick(productId, oneClickUrl, isProductAddtocartFormValidate) {
+    window.__OYST__ = window.__OYST__ || {};
+    window.__OYST__.getOneClickURL = function (cb, opts) {
+        opts = opts || {};
+
+        ready(function () {
+            var qty = $$("input[name='qty']")[0].value ? $$("input[name='qty']")[0].value : 1;
```

Comment:
shorter:
```js
var qty = $$("input[name='qty']")[0].value || 1
```

Reply:
but be careful of js falsies

Reply:
@jney your code set qty to 1 if original qty is 0